iOS-Email-Client/Libs/Env.swift

```diff
@@ -63,21 +63,21 @@ struct Env {
         guard !Env.isProduction else {
             return "criptext.com"
         }
-        return "criptext.com"
+        return "jigl.com"
```

Comment:
PR should not have staging vars
```suggestion
        return "criptext.com"
```